common/app/experiments/Experiments.scala

```diff
@@ -26,6 +28,15 @@ object DCRJSBundleVariant
       participationGroup = Perc1A,
     )
 
+object NoMoreThanFiveSlides
+    extends Experiment(
+      name = "no-more-than-five-slides",
+      description = "DCR bundle experiment",
```

Comment:
has this description come over from a different test?

Reply:
Whoops!